src/algorithms/LAGraph_bfs_pushpull.c

```diff
@@ -617,7 +617,7 @@ GrB_Info LAGraph_bfs_pushpull   // push-pull BFS, or push-only if AT = NULL
 				}
 
 				GxB_Vector_import_CSC(&q, int_type, n,
-						&qi, (void **) (&qx), qi_size, qx_size, iso, nq,
+						&qi, (void **) (&qx), qi_size, qx_size, false, nq,
```

Comment:
This doesn't make sense, if `iso` was set to true by `GxB_Vector_export_CSC` it should be imported as such.

Reply:
You are changing the vector's values between the export and import stage so a vector that was isomorphic might not be isomorphic anymore.

Reply:
You're right, sorry I've missed this.